test/setup/versions.tf

```diff
@@ -15,15 +15,15 @@
  */
 
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.0.0"
```

Comment:
we should not explicitly constrain this to 1.0 unless there is a known incompatibility with lower versions.

Reply:
I updated to the latest version of all the providers, and they don't support Terraform 0.13 which is ancient by now.